### DISCUSSION TITLE
@routes return http endpoint instead of https

### DISCUSSION THREAD

**Discussion by jonaspauleta** [tighten/ziggy#543] [Q&A]
Hello,

I couldn't find any other place to ask a question.
So, I have read the documentation and this is still not clear for me...
On app.blade.php the @routes directive is returning http endpoints instead of https endpoints, even when my APP_URL param in .env file have a https endpoint.
This issue is really baffling me.
Note that I'm using a fresh Laravel Jetstream installation with the command "laravel new demo --jet --stack=inertia".
Hope somebody can help me with this issue, I belive many more people can have this issue on production, but haven't found anything on the web.

Thanks,
João.

**Comment by entity** [comment]
I’m using HTTPS in my project and it works fine.

If you are in production, try run the following;
> PHP artisan optimize
> npm run production

**Comment by bakerkretzmar** [comment]
Are you seeing this locally or on staging/production? Or both?

If your app is using a load balancer or proxy make sure you've configured the trusted proxies middleware—see https://github.com/tighten/ziggy/issues/410#issuecomment-814485042 and [the docs](https://laravel.com/docs/9.x/requests#configuring-trusted-proxies).
